src/client/index.tsx

```diff
@@ -16,87 +16,45 @@ __webpack_public_path__ = getPublicPath();
 import { nbformat } from '@jupyterlab/coreutils';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import { NotebookOutputEventParams } from 'vscode-notebook-renderer';
-import { JupyterNotebookRenderer } from './constants';
 import { CellOutput } from './render';
+import { ActivationFunction, CellInfo } from 'vscode-notebook-renderer';
 
-const notebookApi = acquireNotebookRendererApi(JupyterNotebookRenderer);
-
-notebookApi.onDidCreateOutput(renderOutput);
-
-// Copy of vscode-notebook-renderer old types as of 1.48
-// Keep these so we can support both the old interface and the new interface
-// Interface change here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/51675/files
-interface OldNotebookCellOutputMetadata {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    custom?: { [key: string]: any };
-}
-interface OldNotebookOutput {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    data: { [mimeType: string]: any };
-    metadata?: OldNotebookCellOutputMetadata;
-}
-interface OldNotebookOutputEventParams {
-    element: HTMLElement;
-    outputId: string;
-    output: OldNotebookOutput;
-    mimeType: string;
-}
+export const activate: ActivationFunction = () => ({
```

Comment:
Actual code change was super simple. Webpack was the only real pain. At one point we used to maintain back compat on this extension, hence the old API classes here. But I just don't see that being very possible with the new changes, so explicitly moving to 1.57 and removing the old BC code path.